OmarEspinosaCastro/30-Agosto/Map.java

```diff
@@ -0,0 +1,27 @@
+
+package collection;
+
+public class Map {
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int lengthDirectory = in.nextInt();
+        in.nextLine();
+        HashMap<String, Integer> directorio = new HashMap<String, Integer>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices

directorio?? what's that?? only english please